ProjectSettings/TagManager.asset

```diff
@@ -3,7 +3,8 @@
 --- !u!78 &1
 TagManager:
   serializedVersion: 2
-  tags: []
+  tags:
+  - Background
```

Comment:
It seems like this tag is unneeded, so I think it can be removed.

Reply:
Got it, unneeded tag removed.